tm2/pkg/sdk/options.go

```diff
@@ -34,16 +29,6 @@ func SetMinGasPrices(gasPricesStr string) func(*BaseApp) {
 	return func(bap *BaseApp) { bap.setMinGasPrices(gasPrices) }
 }
 
-// SetHaltHeight returns a BaseApp option function that sets the halt block height.
-func SetHaltHeight(blockHeight uint64) func(*BaseApp) {
```

Comment:
why removing this?

Reply:
This was unused code, not sure how it wasn't caught by the linter